src/libraries/System.Threading.RateLimiting/ref/System.Threading.RateLimiting.cs

```diff
@@ -0,0 +1,83 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// ------------------------------------------------------------------------------
+// Changes to this file must follow the https://aka.ms/api-review process.
+// ------------------------------------------------------------------------------
+
+namespace System.Threading.RateLimiting
+{
+    public sealed partial class ConcurrencyLimiter : System.Threading.RateLimiting.RateLimiter
```

Comment:
Is it odd that we have `RateLimiter`, `TokenBucketRateLimiter`, but then `ConcurrencyLimiter`? Why not `ConcurrencyRateLimiter`?